docs/guides/sdks/ios-sdk/send-consent-dma-ios.md

```diff
@@ -91,51 +91,53 @@ class AppDelegate: UIResponder, UIApplicationDelegate , AppsFlyerLibDelegate {
 
 If your app does not use a CMP compatible with TCF v2.2, use the SDK API detailed below to provide the consent data directly to the SDK.
 
-1. Initialize the SDK in `didFinishLaunchingWithOptions` as described [here](https://dev.appsflyer.com/hc/docs/integrate-ios-sdk#initializing-the-ios-sdk).
-2. In the`applicationDidBecomeActive` lifecycle method determine whether the GDPR applies or not to the user. 
-
-### When GDPR applies to the user
+To manually collect consent data, perform the following:
 
-If GDPR applies to the user, perform the following: 
+1. Initialize the SDK in `didFinishLaunchingWithOptions` as described [here](https://dev.appsflyer.com/hc/docs/integrate-ios-sdk#initializing-the-ios-sdk).
+2. In the`applicationDidBecomeActive` lifecycle method determine whether the GDPR applies or not to the user.
+3. Determine whether the consent data is already stored for this session.
+    - If there is no consent data stored, show the consent dialog to capture the user consent decision.
+    - If there is consent data stored - continue to the next step.
+4. To transfer the consent data to the SDK create an object called [`AppsFlyerConsent`](https://dev.appsflyer.com/hc/docs/ios-send-consent-for-dma-compliance)  with the following parameters:
+    - `isUserSubjectToGDPR` - Indicates whether GDPR applies to the user.
+    - `hasConsentForDataUsage` - Indicates whether the user has consented to use their data for advertising purposes.
+    - `hasConsentForAdsPersonalization` - Indicates whether the user has consented to use their data for personalized advertising.
+    - `hasConsentForAdStorage` - indicates whether the user has consented to store or access information on a device.
 
-1. Given that GDPR is applicable to the user, determine whether the consent data is already stored for this session.
-   1. If there is no consent data stored, show the consent dialog to capture the user consent decision. 
-   2. If there is consent data stored continue to the next step.
-2. To transfer the consent data to the SDK create an `AppsFlyerConsent` object with the following parameters:
-   - `forGDPRUserWithHasConsentForDataUsage` - Indicates whether the user has consented to use their data for advertising purposes.
-   - `hasConsentForAdsPersonalization` - Indicates whether the user has consented to use their data for personalized advertising.
-3. Call `setConsentData()`with the `AppsFlyerConsent` object. 
-4. Call `start()`.  
+> 📘 Note
+>   
+> The SDK will omit any parameters from the event that are not provided by the app.
```

Comment:
I would rephrase:
SDK registers only parameters which are explicitly passed to this method